### ISSUE TITLE
Survey course selector is not accessible via screen reader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#784]
Helaine reports the following:

> for the choose your course option, it's not a drop down for me
> if I start typing, it will suggest a course, but I can't then actually select that course
> and if I click on start survey nothing happens

Even if she uses a link with the course already chosen, she cannot proceed:
> I'm seeing a drop down menu [a list box, more precisely], but when I click on it, it doesn't read me any of the options

**Comment by hudakdidit** [comment]
@ragesoss I've killed the custom select jquery plugin we were using. It has known accessibility issues https://github.com/harvesthq/chosen/issues?utf8=%E2%9C%93&q=is%3Aissue+screen+reader

We already had some custom css styling in place for mobile so, so the field will be a native select input which _should_ be accessible. I'll wait to close until we get confirmation.

**Comment by ragesoss** [comment]
Great. I'll have Helaine test it out as soon as it's live.

**Comment by ragesoss** [comment]
@hudakdidit Helaine reports that although the first selector is accessible, she cannot continue further after clicking 'Start Survey'. Even if the "Start" button on the 'current course' view is clicked, the actual survey content that follows is also not accessible via screen reader.

We can leave this aside for now, though; let's move forward on email design and barebones analytics.

**Comment by ragesoss** [comment]
Unless you have any quick ideas that might fix it, that is.

**Comment by hudakdidit** [comment]
@ragesoss bummer! Nothing immediately comes to mind but will return to this as soon as possible.

**Comment by ragesoss** [comment]
The high priority aspect of this is to do a spike and see if it can be fixed with reasonable effort.

If there's no clear path to fix the accessibility, then we should put this on the backlog and move on.

**Comment by thenickcox** [comment]
@ragesoss Can you give me any more info about the tech Helaine is using?

**Comment by ragesoss** [comment]
She uses the JAWS screen reader.

**Comment by thenickcox** [comment]
I believe that 8f7f2c3 will mostly fix this. It looked like Jonathan killed most of the jQuery Chosen plugin, but there were still instances of it throughout surveys. Not sure if that got reverted later or something. Either way, I confirmed that I was able to navigate the course select and one select option in a survey with just my keyboard, and I tried OS X's VoiceOver screen reader, which I'm sure is different from JAWS, but is hopefully similar enough to give an idea. See what you think.

**Comment by ragesoss** [comment]
This is better, but there are still blockers for getting through the survey flow via JAWS. On one of the matrix slider questions, click 'Next' simply does nothing and the survey won't proceed.

**Comment by ragesoss** [comment]
What's left of the issue is tracked by #828. Closing this one, as the headline issue is resolved.
